src/app/home/home/notification/notification.component.html

```diff
@@ -0,0 +1,10 @@
+<ng-container *ngIf="data.type === 'pending'">
+    <div class="flex flex-row gap-8 items-center">
+        <div class="flex-1">
+            Still the request is pending, click <span class="primary cursor-pointer" (click)="handleRoute()" >here</span> to check profile page
```

Comment:
Multi Lingual text required?

Reply:
Still, some more UI implementation is pending, once it is done. Then I will add